src/core/webview/ClineProvider.ts

```diff
@@ -1453,7 +1455,8 @@ export class ClineProvider extends EventEmitter<ClineProviderEvents> implements
 			browserToolEnabled: stateValues.browserToolEnabled ?? true,
 			telemetrySetting: stateValues.telemetrySetting || "unset",
 			showRooIgnoredFiles: stateValues.showRooIgnoredFiles ?? true,
-			maxReadFileLine: stateValues.maxReadFileLine ?? -1,
+			maxReadFileLine: stateValues.maxReadFileLine ?? 500,
```

Comment:
I want to default this to -1 for now - in my experience the partial reads make the model act dumber.
```suggestion
			maxReadFileLine: stateValues.maxReadFileLine ?? 500,
```